operator/Dockerfile.ubi

```diff
@@ -0,0 +1,36 @@
+# Build the manager binary
+FROM golang:1.16.2 as builder
+
+WORKDIR /workspace
+
+# Copy the go source
+COPY . .
+
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
+
+ARG VERSION=${VERSION}
+ARG RELEASE_NUMBER=${RELEASE_NUMBER}
+
+LABEL name="Tailing Sidecar Operator"
+LABEL maintainer="collection@sumologic.com"
+LABEL vendor="sumologic"
+LABEL version=${VERSION}
+LABEL release=${RELEASE_NUMBER}
+LABEL summary="Tailing Sidecar Operator"
+LABEL description="Tailing Sidecar Operator adds streaming sidecar containers which use tailing sidecar image to Pods"
+
+RUN mkdir /licenses && \
+    curl -O https://raw.githubusercontent.com/SumoLogic/tailing-sidecar/release-v0.3/LICENSE && \
+    mv LICENSE /licenses
```

Comment:
AFAIR you can use `ADD` with URLs

Reply:
sth like
`ADD https://raw.githubusercontent.com/SumoLogic/tailing-sidecar/release-v0.3/LICENSE /licenses/LICENSE`

Reply:
Corrected ;)